goth/runner/__init__.py

```diff
@@ -211,24 +231,35 @@ async def _start_nodes(self):
                 container_ports,
             )
 
-        node_names[self.host_address] = "Pytest-Requestor-Agent"
+        node_names[self.host_address] = "docker-host"
```

Comment:
:sauropod:

Reply:
Yep, that's an old one. I've decided to change it when I looked in to `proxy.log` and messages like this didn't look right:
```
2021-08-18 16:11:35+0000 DEBUG    goth.api_monitor.monitor_addon [request] Pytest-Requestor-Agent:agent -> Pytest-Requestor-Agent:daemon: GET /payment-api/v1/requestorAccounts; body:
```
After this change (and another one, in `router_addon.py`) it looks like this:
```
2021-08-20 13:41:46+0000 DEBUG    goth.api_monitor.monitor_addon [request] docker-host:agent -> requestor:daemon: GET /payment-api/v1/requestorAccounts; body:
```